src/Tutorials/GaussianProcessDataGenerator.cs

```diff
@@ -77,6 +97,8 @@ public void Run()
             Thread thread = new Thread(() => DisplayPNG(model));
             thread.SetApartmentState(ApartmentState.STA);
             thread.Start();
+
+            Console.WriteLine("Plotting complete: Generated {0} points with {1} corrupted", numData, (int)System.Math.Ceiling(numData * propCorrupt));
         }
 
         private void DisplayPNG(PlotModel model)
```

Comment:
remove 'var a'
and maybe put the plot path as a constant attribute of the class

Reply:
Not entirely sure what you mean by `var a`?

Reply:
Ah got you, it was the bit below